### PR TITLE
ConfStore: Properties file with empty and commented lines

### PR DESCRIPTION
**Pull request by suryakumar1024**
Handled Empty lines and Commented lines while reading Properties file

below example properties file works
------------------------------------------------------
example.properties
------------------------------------------------------
bridge = {'name': 'Homebridge', 'username': 'CC:22:3D:E3:CE:30'}
name = Homebridge
username = CC:22:3D:E3:CE:30
manufacturer = homebridge.io
#model details
model = homebridge
pin = 031-45-154
port = 51826
lte_type = [{'name': '3g'}, {'name': '4g'}]

------------------------------------------------------

Signed-off-by: suryakumar <suryakumar.kumaravelan@seagate.com>

### CODE REVIEW

File: py-utils/src/utils/kv_store/kv_store_collection.py [hunk 1]
```diff
@@ -232,8 +232,9 @@ def load(self) -> KvPayload:
         with open(self._store_path, 'r') as f:
             try:
                 for line in f.readlines():
-                    key, val = line.rstrip('\n').split('=', 1)
-                    data[key.strip()] = val.strip()
+                    if line.strip() and line.strip()[0] != '#':
```
Comment (ujjwalpl) [inline py-utils/src/utils/kv_store/kv_store_collection.py:235]: Multiple strip calls and also strip(\n) can be avoided with this to be efficient. 

```
for line in f.readlines(): 
    line = line.strip()
    if not line or line[0] == '#': 
        continue
    key, val = line.split('=', 1)
    data[key.strip()] = val.strip()
```

File: py-utils/test/test_kv_store/test_kv_store.py [hunk 2]
```diff
@@ -55,8 +55,11 @@ def setup_and_generate_sample_files():
 
     with open(r'/tmp/example.properties', 'w+') as file:
         sample_config.update(sample_config['bridge'])
+        file.write("# Comments section test")
```
Comment (ujjwalpl) [inline py-utils/test/test_kv_store/test_kv_store.py:58]: We should cover following cases. 
1. "# normal comment" 
2. " # space before comment"
3. "\n" Empty line in the middle
4. "k = v#dd"
5. "Invalid line with # in the middle" should result in error

**Review by ujjwalpl — CHANGES_REQUESTED** [review]

File: py-utils/src/utils/kv_store/kv_store_collection.py (hunk 1 above)
Comment (suryakumar1024) [reply py-utils/src/utils/kv_store/kv_store_collection.py]: Updated in current patch

File: py-utils/test/test_kv_store/test_kv_store.py (hunk 2 above)
Comment (suryakumar1024) [reply py-utils/test/test_kv_store/test_kv_store.py]: Updated in current patch

**Review by ujjwalpl — APPROVED** [review]

**Review by sachinpunadikar — APPROVED** [review]
+1

**Pull request merged**